### ISSUE TITLE
Icon-only button guidance

### ISSUE THREAD

**Issue by mentalblock99** [mosaic-design-system/docs#394]
Write about how and when icon-only buttons should be used. Speak to Zoe about the accessibility. 

- [ ] ~Designed~
- [ ] Documented
- [ ] Developed
- [ ] Completed
